### PR TITLE
Remove translation from 404 page

### PR DESCRIPTION
**Pull request by MarkvanMents**
The 404 translations were filling up the translation allocation - so removing the translations from 404 pages.

### CODE REVIEW

**Comment by MarkvanMents** [comment]
New page looks like this:
image

File: layouts/404.html [hunk 1]
```diff
@@ -12,11 +12,27 @@
...
+                    <!-- Japanese -->
+                    <p class="page-title" style="font-size: 1rem; margin-bottom:.5rem">もう一つのページが謎の形で流される（404）</p>
+                    <p class="body" style="font-size: 0.8rem;">このページは存在しません。ページ上部の検索バーを使用して、探しているドキュメントを見つけてください。</p>
+                    <a href="{{ "/" | relURL }}" class="btn btn-outline-secondary" style="padding-top: .5rem; padding-bottom: .5rem; padding-left: 1rem; padding-right: 1rem; margin-bottom:.5rem;">家に帰ります</a>
+                    <!-- Korean -->
+                    <p class="page-title" style="font-size: 1rem; margin-bottom:.5rem">신비롭게 씻겨 내려가는 또 다른 페이지(404)</p>
+                    <p class="body" style="font-size: 0.8rem;">이 페이지는 존재하지 않습니다. 페이지 상단의 검색창을 사용해 원하는 문서를 찾아보세요.</p>
+                    <a href="{{ "/" | relURL }}" class="btn btn-outline-secondary" style="padding-top: .5rem; padding-bottom: .5rem; padding-left: 1rem; padding-right: 1rem; margin-bottom:.5rem">집으로 돌아가기</a>
+                    <!-- Chinese -->
+                    <p class="page-title" style="font-size: 1rem; margin-bottom:.5rem">又一页神秘被冲走（404）</p>
```
Comment (Luyao-Zhang-1) [inline layouts/404.html]:
```suggestion
                    <p class="page-title" style="font-size: 1rem; margin-bottom:.5rem">又一个被神秘冲走的页面（404）</p>
```

File: layouts/404.html [hunk 2]
```diff
@@ -12,11 +12,27 @@
...
+                    <p class="body" style="font-size: 0.8rem;">T此页面不存在。请尝试使用页面顶部的搜索栏来查找您正在寻找的文档。</p>
```
Comment (Luyao-Zhang-1) [inline layouts/404.html]:
```suggestion
                    <p class="body" style="font-size: 0.8rem;">此页面不存在。请尝试使用页面顶部的搜索栏来查找您正在寻找的文档。</p>
```

**Comment by koby012** [comment]
I checked Japanese and could you use following ?

1st sentence 
ページがどこかにいってしましました

button
ホームに戻る

**Comment by Luyao-Zhang-1** [comment]
> I checked Japanese and could you use following ?
> 
> 1st sentence ページがどこかにいってしましました
> 
> button ホームに戻る

I replaced the translations with the ones that you proposed. Thanks a lot!

**Comment by TarigMendix** [comment]
here is my opinion for 404 page that you shared.
1st line : 페이지를 찾을 수 없습니다. (404)
2nd line : 이 페이지는 존재하지 않습니다. 페이지 상단의 검색창을 사용해 원하는 문서를 찾아보세요.
button : 홈화면으로 돌아가기 (edited)

**Comment by MarkvanMents** [comment]
Thanks everyone for your input. Unfortunately the design isn't approved so I will revert the 404 page for now and look at incorporating your text updates in a new version of the 404 page in the future.

**Pull request merged**